builtin/core.wren

```diff
@@ -21,6 +21,16 @@ class Sequence {
     }
     return true
   }
+
+  reduce(acc, f) {
+    for (element in this) {
+      acc = f.call(acc, element)
+    }
+    return acc
+  }
+
+  reduce(f) { this[1..-1].reduce(this[0], f) }
```

Comment:
Good point, didn't think about how much nicer it would be with the block syntax. 

I've just recently started working with a static language professionally (Scala) so I'm still trying to undo some of the damage that JS did to me! After your comment I poked around at the APIs in Scala and saw that they also put the seed/acc value in the first place, if used.